testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -1171,6 +1172,32 @@ public void testRenameColumn()
         super.testRenameColumn();
     }
 
+    @Test
+    public void testInsertIntoNotNullColumn()
+    {
+        skipTestUnless(hasBehavior(SUPPORTS_CREATE_TABLE) && hasBehavior(SUPPORTS_INSERT_NOT_NULL_COLUMN));
```

Comment:
when `!hasBehavior(...)`, verify that `CREATE TABLE t(a .. NOT NULL)` is rejected as unsupported (is it?)

Reply:
I will send a new PR.